pkg/generate/app/cmd/newapp.go

```diff
@@ -561,7 +552,17 @@ func (c *AppConfig) RunQuery() (*QueryResult, error) {
 
 func (c *AppConfig) validate() (cmdutil.Environment, cmdutil.Environment, error) {
 	var errs []error
-
+	// Environment File where the parameters are stored
+	if len(c.EnvironmentFile) != 0 {
+		varList, err := app.GetEnvironmentVars(c.EnvironmentFile)
+		if err != nil {
+			return nil, nil, fmt.Errorf("Error reading file %v: %v", c.EnvironmentFile, err)
```

Comment:
No, the message should still report the filename and the actual error
encountered.

Ben Parees | OpenShift

On Sep 2, 2016 5:38 PM, "ravisantoshgudimetla" <notifications@github.com>
wrote:

> In pkg/generate/app/cmd/newapp.go
> <https://github.com/openshift/origin/pull/10766#discussion_r77415320>:
>
> > @@ -561,7 +552,17 @@ func (c *AppConfig) RunQuery() (*QueryResult, error) {
> >
> >  func (c *AppConfig) validate() (cmdutil.Environment, cmdutil.Environment, error) {
> >  	var errs []error
> > -
> > +	// Environment File where the parameters are stored
> > +	if len(c.EnvironmentFile) != 0 {
> > +		varList, err := app.GetEnvironmentVars(c.EnvironmentFile)
> > +		if err != nil {
> > +			return nil, nil, fmt.Errorf("Error reading file %v: %v", c.EnvironmentFile, err)
>
> So, you think output should be something like "Error reading environment
> variables from file " and we can ignore the original error thrown?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/10766/files/136ecd0ae1d769bb264b558662c480582f44f7d8#r77415320>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEvl3iJaUYFiBB8iW8ejAg4d1NNpyxTmks5qmJdKgaJpZM4JyyrC>
> .
>